src/config/config.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+module.exports = {
+  production: {
+    use_env_variable: process.env.DATABASE_URL
+  },
+
+  development: {
+    username: process.env.DB_USERNAME_DEV,
+    password: process.env.DB_PASSWORD_DEV,
+    database: process.env.DATABASE_DEV,
+    host: process.env.DATABASE_URL_DEV,
+    port: process.env.PORT || 5432,
+    dialect: 'postgres'
+  },
+
+  test: {
+    username: process.env.DB_USERNAME_TEST,
+    password: process.env.DB_PASSWORD_TEST,
+    database: process.env.DB_DATABASE_TES,
+    host: process.env.DATABASE_URL_TEST,
+    port: process.env.PORT || 5432,
+    dialect: 'postgres'
+  }
+};
```

Comment:
Always add an empty line at the end of file
```suggestion
 };

```

Reply:
I think there is an empty line here kindly check line 28 and advise @Segun-Ogundipe

Reply:
> Always add an empty line at the end of file

@Segun-Ogundipe, thanks for pointing that out. But there is a Line Feed at the end of the line.

Reply:
> I think there is an empty line here kindly check line 28 and advise @Segun-Ogundipe

I added the empty line as a suggestion that as a suggestion

Reply:
> > Always add an empty line at the end of file
> 
> @Segun-Ogundipe, thanks for pointing that out. But there is a Line Feed at the end of the line.

I added the line to suggest it. You should commit the change